components/Carousel/slick.js

```diff
@@ -0,0 +1,265 @@
+import styled from 'styled-components';
+
+const SlickSliderWrapper = styled.section`
+  padding: 8px;
+
+  .slick-loading .slick-list {
+    background: #fff url('./ajax-loader.gif') center center no-repeat;
+  }
+
+  /* Arrows */
+  .slick-prev,
+  .slick-next {
+    font-size: 0;
+    line-height: 0;
+    position: absolute;
+    top: 50%;
+    display: block;
+    width: 24px;
+    height: 24px;
+    padding: 0;
+    -webkit-transform: translate(0, -50%);
+    -ms-transform: translate(0, -50%);
+    transform: translate(0, -50%);
+    cursor: pointer;
+    border: none;
+    outline: none;
+    background: transparent;
+  }
+  .slick-prev:hover,
+  .slick-prev:focus,
+  .slick-next:hover,
+  .slick-next:focus {
+    color: transparent;
+    outline: none;
+    background: transparent;
+  }
+  .slick-prev:hover:before,
+  .slick-prev:focus:before,
+  .slick-next:hover:before,
+  .slick-next:focus:before {
+    opacity: 1;
+  }
+  .slick-prev.slick-disabled,
+  .slick-next.slick-disabled {
+    opacity: 0.25;
+  }
+
+  .slick-prev:before,
+  .slick-next:before {
+    font-size: 20px;
+    line-height: 1;
+    opacity: 0.75;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+
+  .slick-prev {
+    left: -25px;
+  }
+  [dir='rtl'] .slick-prev {
+    right: -25px;
+    left: auto;
+  }
+  .slick-prev:before {
+    content: '←';
+  }
+  [dir='rtl'] .slick-prev:before {
+    content: '→';
+  }
+
+  .slick-next {
+    right: -25px;
+  }
+  [dir='rtl'] .slick-next {
+    right: auto;
+    left: -25px;
+  }
+  .slick-next:before {
+    content: '→';
+  }
+  [dir='rtl'] .slick-next:before {
+    content: '←';
+  }
+
+  /* Dots */
+
+  .slick-dotted.slick-slider {
+    margin-bottom: 30px;
+  }
+
+  .slick-dots {
+    position: absolute;
+    bottom: -25px;
+    display: block;
+    width: 100%;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    text-align: center;
+  }
+
+  .slick-dots li {
+    position: relative;
+    \display: inline-block;
```

Comment:
**question:** is it a hack?

Reply:
This css is from the lib of react-slick. I will verify it.

Reply:
It was a typing error :smile: